specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -1097,17 +1097,1048 @@
+        "operationId": "LargePersonGroupPerson_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largePersonGroupId"
+          },
+          {
+            "name": "body",
+            "description": "Request body for creating new person.",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/NameAndUserDataContract"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns a new personId created.",
+            "schema": {
+              "$ref": "#/definitions/Person"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create new person for large person group example": {
+            "$ref": "./examples/CreateNewLargePersonGroupPerson.json"
+          }
+        }
+      },
+      "get": {
+        "description": "List all persons in a large person group, and retrieve person information (including personId, name, userData and persistedFaceIds of registered faces of the person).",
+        "operationId": "LargePersonGroupPerson_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largePersonGroupId"
+          },
+          {
+            "name": "start",
+            "description": "Starting person id to return (used to list a range of persons).",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "top",
+            "description": "Number of persons to return starting with the person id indicated by the 'start' parameter.",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1000
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns an array of person information that belong to the large person group.",
+            "schema": {
+              "$ref": "#/definitions/Persons"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List persons in person group": {
+            "$ref": "./examples/ListLargePersonGroupPersons.json"
+          }
+        }
+      }
+    },
+    "/largepersongroups/{largePersonGroupId}/persons/{personId}": {
```

Comment:
Just double confirm, the paths are not case sensitive right?

Reply:
Yes, at least the generated .NET SDK works well. I'd like to append a commit to unify them to lower-case.
(BTW, the parameters inside `{}` are actually user-provided variables, rather than part of path.)

Reply:
I bring this up because I saw: "/largepersongroups/{largePersonGroupId}/persons/{personId}/**persistedFaces**/{persistedFaceId}"
For parameters, lowerCamelCase is good enough.

Reply:
Yes, thanks.